receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -73,6 +80,20 @@ func (oci *Receiver) Export(tes agenttracepb.TraceService_ExportServer) error {
 		return err
 	}
 
+	messageChan := make(chan *data.TraceData, 64)
```

Comment:
It feels like the setup of the workers should happen before the call to Export. Or is the intent really to have a channel (and respective workers) per call to Export?

Reply:
I went back and forth on this -- the previous code had does the grouping per connection (per export).. Thinking more on it, I think it is probably better to have the worker pool per receiver as you are suggesting.